### DISCUSSION TITLE
When you are begining the initial test in javascript, not fit well the css blocks

### DISCUSSION THREAD

**Discussion by Dann-Herz** [codewars/codewars.com#2578] [General]
I'm talking about this part 
![Selection_023](https://user-images.githubusercontent.com/69129863/150961250-e1960284-f292-490d-ba4b-aef386b3a605.png)
and it won't let you submit the answer, but if I change the margin of the element below, works well with 
`margin-top: 80px; //  for example`
![Selection_022](https://user-images.githubusercontent.com/69129863/150961591-def0dfc7-23d4-4ad5-a47f-a06a834a145b.png)
It's just a recommendation, I don't know if this is part of the initial test

**Comment by Blind4Basics** [comment]
Hi,

Could you give the platform/OS/device are you using, plz?

**Comment by Blind4Basics** [comment]
(opened as issue [there](https://github.com/codewars/codewars.com/issues/2579))
